### ISSUE TITLE
Index foreing keys / relationships

### ISSUE THREAD

**Issue by royopa** [floriansemm/SolrBundle#105]
See my entity Demanda class below.

How I can add the index to fields $emissor and $fundos (these fields are foreign keys)

```php
/**
 * Demanda
 *
 * @ORM\Table(name="sicde_demanda")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\DemandaRepository")
 * @ORM\HasLifecycleCallbacks()
 *
 * @Solr\Document(repository="AppBundle\Entity\Demanda")
 *
 */
class Demanda extends Uploadable
{
    /**
     * @var integer
     *
     * @ORM\Column(name="CO_DEMANDA", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     *
     * @Solr\Id
     *
     */
    protected $id;

    /**
     * @var Emissor
     *
     * @ORM\ManyToOne(targetEntity="Emissor")
     * @ORM\JoinColumn(name="CO_EMISSOR", referencedColumnName="CO_EMISSOR")
     *
     */
    protected $emissor;

    /**
     * @var string
     *
     * @ORM\Column(name="DE_IDENTIFICACAO", type="string", length=255, nullable=false)
     *
     * @Solr\Field(type="string")
     *
     */
    protected $identificacao;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="DT_EMISSAO", type="date", nullable=false)
     *
     * @Solr\Field(type="date")
     *
     */
    protected $dataEmissao;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="DT_RECEBIMENTO", type="date", nullable=false)
     *
     * @Solr\Field(type="date")
     *
     */
    protected $dataRecebimento;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="DT_LIMITE_RECURSO", type="date", nullable=false)
     *
     * @Solr\Field(type="date")
     *
     */
    protected $dataLimiteRecurso;

    /**
     * @var Multa
     *
     * @ORM\OneToOne(targetEntity="Multa", mappedBy="demanda", cascade={"persist", "remove"})
     *
     */
    protected $multa;

    /**
     * @var ArrayCollection
     *
     * Fundos vinculados a demanda
     *
     * @ORM\ManyToMany(targetEntity="Fundo")
     * @ORM\JoinTable(name="sicde_demanda_fundo",
     *      joinColumns={@ORM\JoinColumn(name="CO_DEMANDA", referencedColumnName="CO_DEMANDA")},
     *      inverseJoinColumns={@ORM\JoinColumn(name="CO_FUNDO", referencedColumnName="CO_FUNDO", unique=false)}
     *      )
     *
     */
    protected $fundos;

    /**
     * @var string
     *
     * @ORM\Column(name="DE_DEMANDA", type="text", nullable=true)
     *
     * @Solr\Field(type="string")
     *
     */
    protected $descricao;

    /**
     * @var DemandaTipo
     *
     * @ORM\ManyToOne(targetEntity="DemandaTipo")
     * @ORM\JoinColumn(name="CO_DEMANDA_TIPO", referencedColumnName="CO_DEMANDA_TIPO")
     *
     */
    protected $tipo;

    /**
     * @var DemandaStatus
     *
     * @ORM\ManyToOne(targetEntity="DemandaStatus")
     * @ORM\JoinColumn(name="CO_DEMANDA_STATUS", referencedColumnName="CO_DEMANDA_STATUS")
     *
     */
    protected $status;

    /**
     * @var ArrayCollection
     *
     * @ORM\ManyToMany(targetEntity="Unidade")
     * @ORM\JoinTable(name="sicde_demanda_unidade",
     *      joinColumns={@ORM\JoinColumn(name="CO_DEMANDA", referencedColumnName="CO_DEMANDA")},
     *      inverseJoinColumns={@ORM\JoinColumn(name="CO_UNIDADE", referencedColumnName="CO_UNIDADE", unique=false)}
     *      )
     */
    protected $unidadesResponsaveis;
```

**Comment by floriansemm** [comment]
Index Foreign-keys or in this case relations to other objects are not supported

**Comment by floriansemm** [comment]
closed with release [1.4](https://github.com/floriansemm/SolrBundle/releases/tag/1.4)
